backend/src/data/repositories/chat-message/chat-message.repository.ts

```diff
@@ -69,6 +76,17 @@ class ChatMessage {
       .execute();
   }
 
+  public async checkHasUnreadMessages(userId: number): Promise<boolean> {
+    const hasUnreadMessages = await this.#ChatMessageModel
+      .query()
+      .select(1)
```

Comment:
add a constant in the class as a static field

Reply:
What constant? Do you mean 1?

Reply:
no, for `1`

something like 

```suggestion
      .select(ChatMessage.NAME_OF_YOUR_CONSTANT)
```